internal/cmd/templatefuncs.go

```diff
@@ -413,6 +413,22 @@ func (c *Config) quoteTemplateFunc(list ...any) string {
 	return strings.Join(ss, " ")
 }
 
+func (c *Config) squoteTemplateFunc(list ...any) string {
```

Comment:
Would it not be better to implement a single function that `quote`, `quoteList`, `squote`, and `squoteList` (if the latter exists) all use to use the appropriate quoting?